packages/analytics-js-integrations/src/integrations/Braze/browser.js

```diff
@@ -57,6 +57,7 @@ class Braze {
       baseUrl: this.endPoint,
       allowUserSuppliedJavascript: this.allowUserSuppliedJavascript,
     });
+    window.braze.addSdkMetadata([window.braze.BrazeSdkMetadata.CDN, 'rud']);
```

Comment:
@davidbielik can you please share some documentation for this?

Reply:
There is no documentation.
https://github.com/rudderlabs/rudder-sdk-js/pull/1949#discussion_r1875179889

Reply:
we have https://js.appboycdn.com/web-sdk/latest/doc/classes/braze.brazesdkmetadata.html but as you can see Rudderstack hasn't been added as an enum yet.